tools/update_all.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -x -o errexit
+
+tools/generate.py Adw 1 -o src/gi-stubs/repository/Adw.pyi
+tools/generate.py AppIndicator3 0.1 -o src/gi-stubs/repository/AppIndicator3.pyi
+tools/generate.py AppStream 1.0 -o src/gi-stubs/repository/AppStream.pyi
+tools/generate.py Atk 1.0 -o src/gi-stubs/repository/Atk.pyi
+tools/generate.py AyatanaAppIndicator3 0.1 -o src/gi-stubs/repository/AyatanaAppIndicator3.pyi
+tools/generate.py Farstream 0.2 -o src/gi-stubs/repository/Farstream.pyi
+tools/generate.py Flatpak 1.0 -o src/gi-stubs/repository/Flatpak.pyi
+tools/generate.py Gdk 3.0 -o src/gi-stubs/repository/_Gdk3.pyi
+tools/generate.py Gdk 4.0 -o src/gi-stubs/repository/_Gdk4.pyi
+tools/generate.py GdkPixbuf 2.0 -o src/gi-stubs/repository/GdkPixbuf.pyi
+tools/generate.py Geoclue 2.0 -o src/gi-stubs/repository/Geoclue.pyi
+tools/generate.py Ggit 1.0 -o src/gi-stubs/repository/Ggit.pyi
+tools/generate.py Gio 2.0 -o src/gi-stubs/repository/Gio.pyi
+tools/generate.py GIRepository 2.0 -o src/gi-stubs/repository/GIRepository.pyi
+tools/generate.py GLib 2.0 -o src/gi-stubs/repository/GLib.pyi
+tools/generate.py GModule 2.0 -o src/gi-stubs/repository/GModule.pyi
+tools/generate.py Goa 1.0 -o src/gi-stubs/repository/Goa.pyi
+tools/generate.py GObject 2.0 -o src/gi-stubs/repository/GObject.pyi
+tools/generate.py Graphene 1.0 -o src/gi-stubs/repository/Graphene.pyi
+tools/generate.py Gsk 4.0 -o src/gi-stubs/repository/Gsk.pyi
+tools/generate.py GSound 1.0 -o src/gi-stubs/repository/GSound.pyi
+tools/generate.py Gspell 1 -o src/gi-stubs/repository/Gspell.pyi
+tools/generate.py Gst 1.0 -o src/gi-stubs/repository/Gst.pyi
+tools/generate.py GstPbutils 1.0 -o src/gi-stubs/repository/GstPbutils.pyi
```

Comment:
This crashes for me locally.

I managed to trim down the crasher to:

```python3
import gi
gi.require_version("GstPbutils", "1.0")
from gi.repository import GstPbutils
GstPbutils.EncodingAudioProfile()
```

Not sure if this is GStreamer bug, pygobject bug or something we messed up in packaging.

<details>
<summary>Stack trace</summary>

```pascal
#0  g_object_class_install_properties (oclass=0x16b5c80, n_pspecs=3, pspecs=0x7fb493005150 <_properties>) at ../gobject/gobject.c:907
#1  0x00007fb493bfeb0e in type_class_init_Wm (pclass=<optimized out>, node=0x17281a0) at ../gobject/gtype.c:2365
#2  g_type_class_ref (type=<optimized out>) at ../gobject/gtype.c:3080
#3  0x00007fb493bfebc7 in g_type_class_ref (type=type@entry=23625856) at ../gobject/gtype.c:3072
#4  0x00007fb493dbe665 in pygobject_init (self=0x7fb493f42040, args=<optimized out>, kwargs=0x0) at ../gi/pygobject-object.c:1334
#5  0x00007fb4947af6f7 in type_call (type=<optimized out>, args=0x7fb494bcbff8 <_PyRuntime+58904>, kwds=0x0) at Objects/typeobject.c:1103
#6  0x00007fb4948023ed in _PyObject_MakeTpCall (tstate=0x7fb494be6398 <_PyRuntime+166328>, callable=0x16b58c0, args=<optimized out>, nargs=0, keywords=0x0) at Objects/call.c:214
#7  0x00007fb4946f78dd in _PyEval_EvalFrameDefault (tstate=<optimized out>, frame=<optimized out>, throwflag=<optimized out>) at Python/ceval.c:4769
#8  0x00007fb49492c56c in _PyEval_EvalFrame (throwflag=0, frame=0x7fb4942f2020, tstate=0x7fb494be6398 <_PyRuntime+166328>) at ./Include/internal/pycore_ceval.h:73
#9  _PyEval_Vector (tstate=tstate@entry=0x7fb494be6398 <_PyRuntime+166328>, func=func@entry=0x7fb4943f2200, locals=locals@entry=0x7fb49440fd40, kwnames=0x0, argcount=0, args=0x0) at Python/ceval.c:6434
#10 0x00007fb49492c70a in PyEval_EvalCode (co=0x7fb49437d230, globals=<optimized out>, locals=0x7fb49440fd40) at Python/ceval.c:1148
#11 0x00007fb494957430 in run_eval_code_obj (locals=0x7fb49440fd40, globals=0x7fb49440fd40, co=0x7fb49437d230, tstate=0x7fb494be6398 <_PyRuntime+166328>) at Python/pythonrun.c:1710
#12 run_mod (mod=mod@entry=0x152c950, filename=filename@entry=0x7fb49436b7b0, globals=globals@entry=0x7fb49440fd40, locals=locals@entry=0x7fb49440fd40, flags=flags@entry=0x7ffd42596558, 
    arena=arena@entry=0x7fb494333070) at Python/pythonrun.c:1731
#13 0x00007fb494975fc6 in pyrun_file (flags=0x7ffd42596558, closeit=1, locals=0x7fb49440fd40, globals=0x7fb49440fd40, start=257, filename=0x7fb49436b7b0, fp=0x144df70) at Python/pythonrun.c:1626
#14 _PyRun_SimpleFileObject (fp=0x144df70, filename=0x7fb49436b7b0, closeit=1, flags=0x7ffd42596558) at Python/pythonrun.c:440
#15 0x00007fb494976851 in _PyRun_AnyFileObject (fp=0x144df70, filename=0x7fb49436b7b0, closeit=1, flags=0x7ffd42596558) at Python/pythonrun.c:79
#16 0x00007fb49497916f in pymain_run_file_obj (skip_source_first_line=0, filename=0x7fb49436b7b0, program_name=0x7fb4943ae930) at Modules/main.c:360
#17 pymain_run_file (config=0x7fb494bcc3e0 <_PyRuntime+59904>) at Modules/main.c:379
#18 pymain_run_python (exitcode=0x7ffd42596554) at Modules/main.c:601
#19 Py_RunMain () at Modules/main.c:680
#20 0x00007fb49443f0ce in __libc_start_call_main (main=main@entry=0x401040 <main>, argc=argc@entry=2, argv=argv@entry=0x7ffd42596768) at ../sysdeps/nptl/libc_start_call_main.h:58
#21 0x00007fb49443f189 in __libc_start_main_impl (main=0x401040 <main>, argc=2, argv=0x7ffd42596768, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7ffd42596758)
    at ../csu/libc-start.c:360
#22 0x0000000000401075 in _start ()
```

</details>

Reply:
for whatever reason you need to call Gst.init() first

```
import gi
gi.require_version("GstPbutils", "1.0")
from gi.repository import Gst
from gi.repository import GstPbutils

Gst.init()
GstPbutils.EncodingAudioProfile()
```